Dependencies/AndroidExtensions/Source/Globals.cpp

```diff
@@ -7,6 +7,7 @@ namespace android::global
     {
         JavaVM* g_javaVM{};
         jobject g_appContext{};
+        jobject g_currentActivity = nullptr;
```

Comment:
`jobject g_currentActivity` should initialize this to nullptr and be consistent with the other variable initialization.

Reply:
> jobject g_mainActivity{};

Does this not initialize to nullptr by default?

Reply:
Fixed